scripts/tests/test_validate_docstrings.py

```diff
@@ -502,6 +514,33 @@ def no_punctuation(self):
         return "Hello world!"
 
 
+class BadExamples(object):
```

Comment:
I believe there is already an example that imports both of these:

https://github.com/pandas-dev/pandas/blob/5aff7271c23e3a2db1e77bffacf4154b06da1729/scripts/tests/test_validate_docstrings.py#L334

Any reason we don't leverage that instead of creating a new class?